### PR TITLE
feat(DTFS2-6745): remove feature flag

### PR DESCRIPTION
**Pull request by AlexBramhill**
# Introduction
We have been developing 2fa behind a feature flag, this needs to be removed

# Implimentation
Removed feature flag

### CODE REVIEW

File: portal-api/api-tests/api-test-users.js [hunk 1]
```diff
@@ -232,18 +231,9 @@ const apiTestUser = {
...
-    return { userId, token };
-  }
-
+const loginTestUser = async (as, user) =>
   // Users are fully logged in by default
-  return createLoggedInUserSession(user);
-};
-
+  createLoggedInUserSession(user);
 const setUpApiTestUser = async (as) => {
```
Comment (lmarrai-sw) [inline portal-api/api-tests/api-test-users.js:237]:
```suggestion
  createLoggedInUserSession(user);
  
const setUpApiTestUser = async (as) => {
```

**Review by lmarrai-sw — COMMENTED** [review]
Thanks for this @AlexBramhill, just 1 comment from me (above)
We need to discuss with @abhi-markan how we want to merge the 2fa work before we merge this though!

File: portal/server/controllers/login/check-your-email.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ module.exports.renderCheckYourEmailPage = (req, res) => {
     case 0:
       return res.render('login/we-have-sent-you-another-link.njk', { obscuredSignInLinkTargetEmailAddress: obscureEmail(userEmail) });
     case -1:
-      return res.render('login/temporarily-suspended.njk');
+      return res.status(403).render('login/temporarily-suspended.njk');
```
Comment (AlexBramhill) [inline portal/server/controllers/login/check-your-email.js:27]: I've added this to return a 403 as there's existing precident for returning correct statuses on failed requests (ie delete account)

File: portal/server/routes/login/index.js [hunk 3]
```diff
@@ -79,7 +57,8 @@ router.post('/login', async (req, res) => {
       req.session.numberOfSendSignInLinkAttemptsRemaining = numberOfSendSignInLinkAttemptsRemaining;
     } catch (sendSignInLinkError) {
       if (sendSignInLinkError.response?.status === 403) {
-        return res.render('login/temporarily-suspended.njk');
+        req.session.numberOfSendSignInLinkAttemptsRemaining = -1;
+        return res.status(403).render('login/temporarily-suspended.njk');
```
Comment (AlexBramhill) [inline portal/server/routes/login/index.js:61]: I've added this to return a 403 as there's existing precident for returning correct statuses on failed requests (ie delete account)

File: portal/server/routes/login/index.js [hunk 4]
```diff
@@ -88,7 +67,7 @@ router.post('/login', async (req, res) => {
     console.warn('Failed to login: %O', loginError);
 
     if (loginError.response?.status === 403) {
-      return res.render('login/temporarily-suspended.njk');
+      return res.status(403).render('login/temporarily-suspended.njk');
```
Comment (AlexBramhill) [inline portal/server/routes/login/index.js:70]: I've added this to return a 403 as there's existing precident for returning correct statuses on failed requests (ie delete account)

**Review by lmarrai-sw — DISMISSED** [review]
Thanks @AlexBramhill

**Review by lmarrai-sw — APPROVED** [review]

**Review by lmarrai-sw — DISMISSED** [review]

**Review by lmarrai-sw — APPROVED** [review]

**Review by abhi-markan — APPROVED** [review]

**Pull request merged**